tests/test_app.py

```diff
@@ -117,6 +119,81 @@ def test_add_command_accepted(chalice_app):
     assert "OK, hang on" in ri["slack_message"][1]["json"]["text"]
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("date", ["today", "2020-01-01"])
+def test_add_command_accepted_and_edited(chalice_app, date):
+    user_id = f"{random.randint(0, 10000)}"
+    r = call_from_slack(
+        chalice_app=chalice_app,
+        full_command=f"add vab {date} 8",
+        user_id=user_id,
+        user_name="mattias",
+    )
+
+    assert r["response"]["statusCode"] == 200
+    assert r["slack_message"][1]["json"]["text"] == "From timereport"
+    attachments = r["slack_message"][1]["json"]["attachments"]
+    assert attachments is not None
+    attachment = attachments[0]
+    assert len(attachment["actions"]) > 0
```

Comment:
```suggestion
    assert attachment["actions"]
```

Could also add assertion about that the data type should be a list